src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -143,6 +148,30 @@ private protected async Task CheckResultsAsync(
             List<CompletionFilter> matchingFilters,
             CompletionItemFlags? flags,
             CompletionOptions options)
+        {
+            var expectedResult = new CompletionTestExpectedResult(
+                Name: expectedItemOrNull,
+                IsAbsent: checkForAbsence,
+                ExpectedDescription: expectedDescriptionOrNull,
+                Glyph: glyph,
+                MatchPriority: matchPriority,
+                DisplayTextSuffix: displayTextSuffix,
+                DisplayTextPrefix: displayTextPrefix,
+                InlineDescription: inlineDescription,
+                IsComplexTextEdit: isComplexTextEdit);
+
+            var expectedResultArray = new[] { expectedResult };
```

Comment:
Can you inline that with collection expression? Or that would result in an overload ambiguity?